scripts/storage-emulator-integration/utils.ts

```diff
@@ -66,25 +38,41 @@ export function getAuthEmulatorHost(emulatorConfig: FrameworkOptions) {
   throw new Error("Auth emulator config not found or invalid");
 }
 
-export function getStorageEmulatorHost(emulatorConfig: FrameworkOptions) {
-  const port = emulatorConfig.emulators?.storage?.port;
-  if (port) {
-    return `http://localhost:${port}`;
+/**
```

Comment:
Would be good to consolidate the code in this file with the one in the new env.ts file in general. I see many functions doing the same things

Reply:
done